tests/core/test_batch_definition.py

```diff
@@ -63,6 +66,73 @@ def test_build_batch_request(
     )
 
 
+@pytest.mark.unit
+def test_get_batch_retrieves_only_batch(mocker: pytest_mock.MockFixture):
```

Comment:
Some duplication between these tests but writing one test and parameterizing felt a bit ugly